azure-container-apps/ReadMe.md

```diff
@@ -1,337 +1,349 @@
+2. Edit the following in the .json file you downloaded:
+3. 1. **Actor**: Enter the email address of the administrator in Google Workspace that the service account is acting on behalf of.
+	2. **Bridge Address**: Enter your SCIM bridge domain (not your 1Password account sign-in address). This is the Application URL for your Container App found on the overview page. For example: `https://scim.example.com`.
+4. Save the file.
+5. Run the following command for your shell to create a base64 value for the file.
+    - **Bash**:
+    ```bash
+    cat /path/to/workspace-settings.json | base64
+    ```
+    - **Using PowerShell**:
+    ```pwsh
+    [Convert]::ToBase64String([IO.File]::ReadAllBytes((Join-Path $PWD.Path '/path/to/workspace-settings.json')))
+    ```
+
+#### Step 3: Add the secrets to your container
+
+1. Choose **Secrets** from the Settings section in the sidebar.
+2. Click **Add**, then fill out the following fields:
+     1. **Key**: Enter `workspace-settings`.
+     2. **Value**: Enter the base64 value you just created.
+
+3. Click **Add**, then wait until the secret is created.
+4. Choose **Containers** from Application section in the sidebar.
+5. Click **Edit and deploy**.
+6. Hover over your **op-scim-bridge** container and select it, then click **Edit**.
+7. Scroll down to the environment variables and add a new one. Then fill out the following fields:
+      1. **Name**: Enter `OP_WORKSPACE_CREDENTIALS`.
+      2. **Source**: Choose **Reference a secret**.
+      3. **Value**: Select the `workspace-credentials` secret.
+
+8. Create another new variable and fill out the following fields:
+      1. Name: **OP_WORKSPACE_SETTINGS**
+      2. **Source**: Choose **Reference a secret**.
+      3. **Value**: Select the `workspace-settings` secret.
+
+9. Click **Save**, then click **Create**.
 
 </details>
 
 <hr>
 
-## Update 1Password SCIM Bridge
+## Update your SCIM bridge
+
+
+> [!TIP]
+>
+> Check for 1Password SCIM Bridge updates on the [SCIM bridge release page](https://app-updates.agilebits.com/product_history/SCIM).
 
-#### Update using the Azure Cloud Shell or AZ CLI
-The latest version of 1Password SCIM Bridge is posted on our [Release Notes](https://app-updates.agilebits.com/product_history/SCIM) website, where you can find details about the latest changes. 
+You can update your SCIM bridge in the [Azure Cloud Shell](#in-the-azure-cloud-shell-or-az-cli) or the [Azure Portal](#in-the-azure-portal).
 
-1. Start the Azure Cloud Shell from the navigation bar of your [Azure Portal](https://portal.azure.com) or directly open the [Azure Shell](https://shell.azure.com) or use the `az` CLI tool.
-2. Run the following command, replacing `$ConAppName` and `$ResourceGroup` with the names from your deployment. Also ensure to change the version number 2.9.0 to match the latest version from our [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM).
+### In the Azure Cloud Shell or AZ CLI
```

Comment:
Do we remove this section about updating in the Azure Cloud Shell or AZ CLI and point to our https://support.1password.com/scim-update/#azure-container-apps guide?

Reply:
Yes! I'll do that.